### DISCUSSION TITLE
Vercel dev command to support more arguments

### DISCUSSION THREAD

**Discussion by AnishDe12020** [vercel/vercel#6544] [Ideas] [answered]
Yarn and NPM can pass on arguments to the respective scripts with the `--` operator. For example if I want to run my project at port 3001 instead of port 3000, I can run `yarn dev -- -p 3001` and the project would run at the specified port. So any news of this feature coming to the Vercel CLI?

**Comment by leerob** [comment, marked as answer]
`vercel dev --listen 3001`
